src/View/HelperTrait.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Zend\I18n\View;
+
+use IntlDateFormatter;
+
+// @codingStandardsIgnoreStart
```

Comment:
Here is the start, where is the end?

Reply:
Added it at the bottom, below the trait.